src/utils/domUtils.ts

```diff
@@ -127,6 +127,11 @@ export module Utils {
       }
       return null; // not in the DOM
     }
+
+    var _latestClipPathId = 0;
+    export function getUniqueClipPathId() {
+      return ++_latestClipPathId;
```

Comment:
Maybe this should return a `string` that includes a prefix so it can be used directly in `_generateClipPath()`.